test/frontend/LoginTest.py

```diff
@@ -0,0 +1,50 @@
+import unittest
```

Comment:
In general please add a notes.md file to the `tests/frontend` folder with a description all install requirements and how the tests can be executed.